.github/workflows/test.yml

```diff
@@ -603,12 +605,12 @@ jobs:
       - name: Build nextest
         timeout-minutes: 120
         run: |
-          cargo nextest run --no-run --workspace --release --exclude turbo --exclude turborepo-ffi --exclude turborepo-lib
+          cargo nextest run --no-run --workspace --release --exclude turbo --exclude turborepo-ffi --exclude turborepo-lib --exclude turborepo-scm
```

Comment:
Why are we excluding this crate? If I understand we exclude `turbo` and `turborepo-ffi` to skip setting up Go/Protobuf stuff and we skip `turborepo-lib` due to the native vs rust tls choice. If we're excluding it here then should it be added to the `turborepo_rust_test` action?

Reply:
Yeah I need to add it there. We exclude all turborepo crates so that turbopack doesn't run any of our tests.